spec/lib/tasks/forms.rake_spec.rb

```diff
@@ -147,19 +140,15 @@
       let(:submission_email) { "test@example.gov.uk" }
       let(:valid_args) { [form.id, submission_email] }
 
-      before do
-        allow(FormRepository).to receive_messages(find: form, save!: form)
-      end
-
       shared_examples "submission email update" do
         it "changes the form submission email" do
-          expect(FormRepository).to receive(:save!).with(an_object_having_attributes(submission_email:))
-          task.invoke(*valid_args)
+          expect {
+            task.invoke(*valid_args)
+          }.to change { form.reload.submission_email }.to(submission_email)
         end
 
         it "updates the email confirmation status" do
-          expect(FormRepository).to receive(:save!).with(an_object_having_attributes(email_confirmation_status: :email_set_without_confirmation))
-          task.invoke(*valid_args)
+          expect(form.reload.email_confirmation_status).to eq(:email_set_without_confirmation)
```

Comment:
This shared example no longer calls `task.invoke` anywhere, is that right?

Reply:
Good spot, have updated